src/gc/unix/configure.cmake

```diff
@@ -54,4 +54,4 @@ check_cxx_source_runs("
 
 check_library_exists(c sched_getaffinity "" HAVE_SCHED_GETAFFINITY)
 
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/config.h.in ${CMAKE_CURRENT_BINARY_DIR}/config.h)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/unix/config.h.in ${CMAKE_CURRENT_BINARY_DIR}/config.h)
```

Comment:
I'd prefer replacing `CMAKE_CURRENT_SOURCE_DIR` by `CMAKE_CURRENT_LIST_DIR` here instead of adding the `/unix/`.